composer.json

```diff
@@ -22,7 +22,10 @@
         "phpunit/phpunit": "^4.7"
     },
     "autoload": {
-        "psr-0": { "WindowsAzure\\": "" }
+        "psr-0": { "WindowsAzure\\": "WindowsAzure/" }
```

Comment:
No negative impact since you're not using old PEAR-style pseudo namespacing: http://stackoverflow.com/a/24869629. Indeed, it would be nice to rename the directory `WindowsAzure` to `src`.